Source/Http/RequestParameter.swift

```diff
@@ -21,30 +21,36 @@
 import Foundation
 
 struct RequestParameter {
-    private var storage: [String: AnyObject] = [:]
+    private var storage: [String: Any] = [:]
     
-    init(_ parameters: [String: Any?] = [:]) {
+    init(_ parameters: [String: Any] = [:]) {
```

Comment:
The original code is for filtering the nil parameter, and give a specific handling for bool type. I think the following code change is a little confused. Can we update it like this? 
    init(_ parameters: [String: Any?] = [:]) {
        for (key, value) in parameters {
            guard let realValue = value else {
                continue
            }
			
            switch realValue {
			case let bool as Bool:
				storage.updateValue(String(bool), forKey: key)
                        default:
                                storage.updateValue(realValue, forKey: key)
            }
        }
    }